quill-sql/src/main/scala/io/getquill/context/sql/idiom/SqlIdiom.scala

```diff
@@ -467,14 +463,35 @@ trait SqlIdiom extends Idiom {
             SqlIdiom.withActionAlias(this, r)
           case ReturningClauseSupported =>
             stmt"${action.token} RETURNING ${returnListTokenizer.token(ExpandReturning(r)(this, strategy).map(_._1))}"
-          case other =>
-            stmt"${action.token}"
+          case OutputClauseSupported => action match {
+            case Insert(entity: Entity, assignments) =>
+              val (table, columns, values) = insertInfo(insertEntityTokenizer, entity, assignments)
+              stmt"INSERT $table${actionAlias.map(alias => stmt" AS ${alias.token}").getOrElse(stmt"")} (${columns.mkStmt(",")}) OUTPUT ${outputs(prop)} VALUES (${values.map(scopedTokenizer(_)).mkStmt(", ")})"
```

Comment:
Are you sure you need to write a new `outputs` function? Maybe you can just use ReturningListTokenizer i.e. do:
```scala
returnListTokenizer.token(ExpandReturning(r, Some("INSERTED"))(this, strategy).map(_._1))
```

Reply:
No I am not sure. Tbh, `ExpandReturning` is exactly where I was expecting some feedback, I didn't know how to change it.